docs/how-to/understanding-break-the-monolith.md

```diff
@@ -27,33 +27,37 @@ If an entity in your original monolith service has a non-relation field like `cr
 
 ### Handling One-to-One Relations
 
-In one-to-one relationships, both entities involved in the relationship retain a single field representing the remote entity's ID. This approach maintains a link between the entities even after they are distributed across different services.
+In one-to-one relationships, both entities involved in the relationship undergo a modification where their related field names are suffixed with `Id`. This approach maintains a link between the entities even after they are distributed across different services.
```

Comment:
Do we want to say anything about the type of the fields (and not only about their names)?
In addition- "This approach maintains a virtual link..." - WDYT? Because it's not a "real" link as we are talking here about two different DBs.